brave/src/main/java/brave/sampler/CountingSampler.java

```diff
@@ -5,7 +5,8 @@
 
 /**
  * This sampler is appropriate for low-traffic instrumentation (ex servers that each receive <100K
- * requests), or those who do not provision random trace ids. It not appropriate for collectors as
+ * requests), or those who do not provision random trace ids. It is not appropriate for collectors
+ * as the sampling decision isn't idempotent (consistent based on trace id).
  * the sampling decision isn't idempotent (consistent based on trace id).
```

Comment:
Well, i guess that feature is in beta....
```suggestion
```

Reply:
I wonder how to do a multiline suggestion?

Reply:
Heh, I thought I would give it a try. Seemed convenient. :)